pkg/virt-handler/vm.go

```diff
@@ -1226,6 +1218,35 @@ func isGuestAgentSupported(vmi *v1.VirtualMachineInstance, commands []v1.GuestAg
 	return true
 }
 
+func calculatePausedCondition(vmi *v1.VirtualMachineInstance, reason api.StateChangeReason) {
+	switch reason {
+	case api.ReasonPausedUser:
+		log.Log.Object(vmi).V(3).Info("Adding paused condition")
```

Comment:
@xpivarc are you sure you want to keep these logs? we will have this event recorded in the VMI status anyway

Reply:
It can help with debugging(It helped me at least) and the update call can fail.